chain/toolsToChain.go

```diff
@@ -46,7 +47,9 @@ func queryOverlap(alpha Chain, beta bed.Bed) bool {
 	}
 }
 
-func ChainToBed(ch Chain, checkTarget bool) bed.Bed {
+// ChainToBed converts a chain into a bed region equal to the target bases covered by the chain,
+// or the query bases if useTarget is false.
+func ChainToBed(ch Chain, useTarget bool) bed.Bed {
 	if checkTarget {
```

Comment:
problem with this variable

Reply:
Thank you for catching this.  I believe I just pushed a fix.